### PR TITLE
changed course to program

### LINKED ISSUE #52: As a User I want to change courses to programs So it is easier to understand.
Acceptance criteria:
GIVEN On the admin panel or blog page,
WHEN Things are related to "course",
THEN Change it into "program".

### PR DESCRIPTION
**Pull request by Koreyoshi159**
Changed all related things to "course" into "program".
On all paged the link to courseController also to the right controller

### CODE REVIEW

File: app/Http/Controllers/ProgramController.php [hunk 1]
```diff
@@ -33,57 +33,57 @@ public function create()
...
+        Program::create($this->getValidate($request));
 
         return redirect(route('admin.index'));
     }
 
     /**
      * Display the specified resource.
      *
-     * @param \App\Models\Course $course
+     * @param  \App\Models\Program  $program
```
Comment (avanheeren) [inline app/Http/Controllers/ProgramController.php]: there are 2 spaces between Program and $program

**Review by avanheeren — CHANGES_REQUESTED** [review]
Nice work, just one space too much

File: app/Models/Program.php [hunk 2]
```diff
@@ -5,7 +5,7 @@
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
 
-class Course extends Model
+class Program extends Model
 {
     use HasFactory;
     protected $guarded = [];
```
Comment (awqfrxx) [inline app/Models/Program.php:11]: we still using guarded? or we changing to fillable

**Review by awqfrxx — APPROVED** [review]

**Review by avanheeren — APPROVED** [review]

**Review by koenhamstra — APPROVED** [review]
Looks good

**Pull request merged**